src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -44,121 +58,181 @@ export function formatDate(date) {
+        </p>
+      );
+      break;
+    case STATUS_TYPES.onDocket:
+      contents.title = 'Waiting to be assigned to a judge';
+      contents.description = (
+        <p>Your appeal is at the Board of Veterans’ Appeals waiting to be
+        assigned to a Veterans Law Judge. Staff at the Board are making sure that your case is
+        complete, accurate, and ready to be decided by a judge. If you have evidence that isn’t
+        already included in your case, now is a good time to send it to the Board.</p>);
+      break;
+    case STATUS_TYPES.pendingCertificationSsoc:
+      contents.title = 'Current Status title';
+      contents.description = (<p>Current Status description</p>);
+      break;
+    case STATUS_TYPES.pendingCertification:
+      contents.title = 'Current Status title';
+      contents.description = (<p>Current Status description</p>);
+      break;
+    case STATUS_TYPES.pendingForm9:
+      contents.title = 'Review your Statement of the Case';
+      contents.description = (
         <div>
-          For issues that are allowed, you will receive compensation. For more information, please
-          contact your VSO or representative.
+          <p>
+            VA sent you a Statement of the Case on [SENT DATE]. The Statement of the Case document
+            explains the reasons why the Regional Office could not fully grant your appeal.
+          </p>
+          <p>
+            If you don’t agree with the Statement of the Case, you can bring your appeal to the Board
+            of Veterans’ Appeals. To do this you must complete and return a Form 9 within [DAYS LEFT TO SUBMIT].
+          </p>
+          <p>
+            <em>Note:</em> If you send more evidence with the Form 9, it will be reviewed by
+            VA and will likely delay a decision on your appeal. If you have new evidence,
+            it’s best to send it when the Board of Veterans’ Appeals is reviewing your case.
+          </p>
         </div>
-      </div>
-    );
-  } else if (type === remand) {
-    const { decisionIssues } = details;
-    const allowedIssues = decisionIssues
-      .filter((issue) => (issue.disposition === 'allowed'))
-      .map((issue, i) => (<li key={`allowed-${i}`}>{issue.description}</li>));
-    const deniedIssues = decisionIssues
-      .filter((issue) => (issue.disposition === 'denied'))
-      .map((issue, i) => (<li key={`denied-${i}`}>{issue.description}</li>));
-    const remandIssues = decisionIssues
-      .filter((issue) => (issue.disposition === 'remand'))
-      .map((issue, i) => (<li key={`remanded-${i}`}>{issue.description}</li>));
-    const businessDays = details.businessDays;
-    contents.title = 'The Board has made a decision on some of your appeals';
-    contents.description = (
-      <div>
+      );
+      break;
+    case STATUS_TYPES.pendingSoc:
+      contents.title = 'Current Status title';
+      contents.description = (<p>Current Status description</p>);
```

Comment:
I think we can add [these](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Appeals%20Status/V2/Design%20files/Statuses.md#pending-soc) titles and descriptions now. Unless you're holding off because the content isn't final, in which case, don't pay me no nevermind.

Reply:
Yep, confirmed with @gnakm that content's finalized now so I'll push that into this PR